### ISSUE TITLE
Update code in doc/ directory to use new application

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10369]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)| @puzzlement reported|
|-|-|
|Trac ID|trac#544|
|Type|defect|
|Created|2004-03-06 07:50:27Z|

<details><summary>Searchable metadata</summary>

```
trac-id__544 544
type__defect defect
reporter__hypatia hypatia
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__documentation documentation
time__1078559427000000 1078559427000000
changetime__1474244394264856 1474244394264856
version__ 
owner__ 
cc__hypatia
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Here's the result of a trivial grep:

howto/listings/manhole/manhole1.py:from twisted.internet.app import Application
howto/listings/manhole/manhole2.py:from twisted.internet.app import Application
howto/listings/manhole/manhole3.py:from twisted.internet.app import Application
howto/listings/pb/copy_receiver.py:from twisted.internet.app import Application
howto/listings/pb/cache_receiver.py:from twisted.internet.app import Application
howto/listings/pb/copy2_receiver.py:from twisted.internet.app import Application
howto/listings/pb/go_server.py:import twisted.internet.app
howto/listings/pb/go_server.py:    app =
twisted.internet.app.Application("go_server")
howto/listings/pb/pb4server.py:import twisted.internet.app
howto/listings/pb/pb4server.py:app = twisted.internet.app.Application("pb4server")
howto/listings/pb/pb7server.py:import twisted.internet.app
howto/listings/pb/pb7server.py:app = twisted.internet.app.Application("pb7server")
howto/listings/pb/pb7server.py:multi =
twisted.internet.app.MultiService("shared", app)

Yuck.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
OK, the manhole examples were removed, and copy_receiver.py, cache_receiver.py,
copy2_receiver.py and pb4server.py updated.

go_server.py and pb7server.py need to be updated to newcred, which probably
means I'll port them to the new PB API at the same time that I port them to new
application.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
go_server and pb7server weren't used by current HOWTOs anyway: removed them.

Marked bug done.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_5"></a>|@rodrigc commented|
|-|-|

In changeset 9481baa4d36d4fb462de5115d3693fb993ac36e1
```
#!CommitTicketReference repository="" revision="9481baa4d36d4fb462de5115d3693fb993ac36e1"
Merge pull request #544 from twisted/8830-mail-py3

Author: berdario
Reviewer: rodrigc
Fixes: #8330

Fix some of the imap tests for Python 3
```
